.github/workflows/mlc_config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "ignorePatterns": [
+    {
+      "pattern": "^\.\.\/v1docs\/"
```

Comment:
Are we ignoring this because we accept the old docs will have dead/outdated links?

Reply:
It's because the folder doesn't exist as part of this CI. The workflow which runs on `v1docs` is different and is in the other branch. We need the link for navigation but have to ignore it for the CI as the folder isn't there in this branch.